.github/workflows/semantic-release.yml

```
@@ -0,0 +1,47 @@
+name: Semantic Release
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)